src/darwin/Framework/CHIP/MTRLogging_Internal.h

```diff
@@ -1,7 +1,6 @@
-/*
+/**
  *
  *    Copyright (c) 2020 Project CHIP Authors
- *    All rights reserved.
```

Comment:
Why this license change?

Reply:
This license header didn't match the ones from other files

Reply:
That really worries me, actually...  Probably best to make this change in a separate PR where we can figure out what the right text is.